Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_V2_0.h

```diff
@@ -24,10 +24,10 @@
 #define SKR_MINI_E3_V2
 
 // Onboard I2C EEPROM
-#undef FLASH_EEPROM_EMULATION //temporary EEPROM include hack
-#undef NO_EEPROM_SELECTED     //temporary EEPROM include hack
-#define I2C_EEPROM
-#define MARLIN_EEPROM_SIZE 0x1000                 // 4KB
+#if NO_EEPROM_SELECTED
+  #define I2C_EEPROM
+  #define MARLIN_EEPROM_SIZE 0x1000                 // 4KB
+#endif
```

Comment:
This change broke my build: `#error "Please select only one method of EEPROM Persistent Storage."`

Reply:
Excellent. I had hoped one of the CI tests would net something.
What form of EEPROM storage are you setting (or not) in your config?

Reply:
I'm flattered that you'd consider my local environment a CI test :-)
- I've pulled latest from this branch
- I'm using `Configuration.h` and `Configuration_adv.h` from the sister configuration PR here: https://github.com/thisiskeithb/Configurations/tree/pr/add_skr_mini_e3_v2/config/examples/BigTreeTech/SKR%20Mini%20E3%202.0
- `default_envs = STM32F103RC_btt` in platformio.ini

Other than that, no changes.

Reply:
Should we also define `I2C_EEPROM` in `Configuration.h`? There isn't a spot for it specifically and the EEPROM is non-removable, so it seems to make the most sense to define it in the V2 pins file.

Reply:
Only define an EEPROM type via configs when it's something other than the hardware default.